src/modules/Lan.rb

```diff
@@ -1039,7 +1042,6 @@ def Packages
         # for wlan require iw instead of wireless-tools (bnc#539669)
         "wlan" => "iw",
         "vlan" => "vlan",
-        "br"   => "bridge-utils",
```

Comment:
As we are using iptools2 and wicked, this dependency should be removed as this way of configure bridges is currently deprecated.